utils/sysinfo/sysinfo.go

```diff
@@ -220,9 +220,12 @@ func GetNodesInfo(sysFs sysfs.SysFs) ([]info.Node, int, error) {
 		}
 
 		allLogicalCoresCount += len(cpuDirs)
+
+		// On some Linux platforms(such as Arm64 guest kernel), cache info may not exist.
+		// So, we should ignore error here.
 		err = addCacheInfo(sysFs, &node)
 		if err != nil {
-			return nil, 0, err
+			klog.Warningf("Found node without cache information, nodeDir: %s", nodeDir)
```

Comment:
Will this appear in the syslog / journal by default? If so, that may be a concern and I would encourage using a lower log level, or a flag to suppress it.

I came here via an issue on the k3s repo after having a log fill up with this message. Turns out, this code gets hit a fair bit.

Reply:
@alexellis 
Unfortunately, this is still in the syslog.
I will resubmit a PR to solve your troubles